feature-libs/asm/components/asm-main-ui/asm-main-ui.component.html

```diff
@@ -13,6 +13,18 @@
     </div>
   </div>
   <div class="asm-bar-actions">
+    <div class="cx-asm-help-portal">
+      <a
+        href="https://help.sap.com/docs/SAP_COMMERCE_COMPOSABLE_STOREFRONT/d1ca9a1517f847a5b57b4e08400807c1/2367ff04ccb649ac82a3795ecf72a804.html?locale=en-US"
+        target="_blank"
+        rel="noopener noreferrer"
+        role="button"
+      >
+        <i class="fa-regular fa-circle-question"></i>
```

Comment:
Where have you defined the class ‘fa-regular’?

Reply:
define the icon named help, which is sourced from fontawesome's far fa-circle-question.